app/javascript/dashboard/components/widgets/AutomationActionTeamMessageInput.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <div class="multiselect-wrap--small">
+      <multiselect
+        v-model="selectedTeams"
+        track-by="id"
+        label="name"
+        :placeholder="'Select Teams'"
+        :multiple="true"
+        selected-label
+        :select-label="$t('FORMS.MULTISELECT.ENTER_TO_SELECT')"
+        deselect-label=""
+        :max-height="160"
+        :options="teams"
+        :allow-empty="false"
+        @input="updateValue"
+      />
+      <textarea
+        v-model="message"
+        rows="4"
+        placeholder="Enter your message here"
+        @input="updateValue"
+      ></textarea>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  // Could
+  props: ['teams', 'value'],
```

Comment:
Prop validation and type would make it more redable

Reply:
@nithindavid  The data type is here actually dynamic for `value`, it can be either a string, object or an array, and vues lint kept throwing type error. Hence removed the validation